ocaml/xapi/context.ml

```diff
@@ -84,6 +84,9 @@ let string_of x =
 
 let database_of x = x.database
 
+let set_database x db =
```

Comment:
This does not mutate `x`, perhaps a different name than `set` would be better (`update`?)

Reply:
I've changed this to update